src/robot_manager.cpp

```diff
@@ -94,8 +100,11 @@ RobotManager::RobotManager()
 
 
   // Read the robot config for this manager.
-  std::string yaml_filename =
-      ros::package::getPath(ROS_PACKAGE_NAME) + "/conf/" + temoto_core::common::getTemotoNamespace() + ".yaml";
+  TEMOTO_INFO_STREAM(temoto_core::common::getTemotoNamespace());
+
+  std::string yaml_filename = 
+      ros::package::getPath(ROS_PACKAGE_NAME) + "/conf/" + "robot_description.yaml";
```

Comment:
Just a reminder, that this section has to be arranged to read configurations from arbitrary paths.